core/src/main/java/io/fabric8/maven/core/config/ProcessorConfig.java

```diff
@@ -31,6 +31,8 @@
 
     public static final ProcessorConfig EMPTY = new ProcessorConfig();
 
+    public static final ProcessorConfig FULL = new ProcessorConfig(null, new TreeSet<String>(), null);
+
```

Comment:
BTW, as far as I see, the `EMPTY` and `FULL` above are doing the same: Including everything. I just introduced a 'INCLUDES_ALL' and 'EXCLUDES_ALL' to make it more easier to understand (although `EXCLUDES_ALL` is not really used).